Common/src/interface_interpolation/CInterpolator.cpp

```diff
@@ -97,15 +97,14 @@ void CInterpolator::Collect_VertexInfo(int markDonor, int markTarget,
     if (donor_geometry->nodes->GetDomain(iPointDonor)) {
       Buffer_Send_GlobalPoint[iLocalVertexDonor] = donor_geometry->nodes->GetGlobalIndex(iPointDonor);
       for (iDim = 0; iDim < nDim; iDim++)
-        Buffer_Send_Coord[iLocalVertexDonor*nDim+iDim] = donor_geometry->nodes->GetCoord(iPointDonor, iDim);
+        Buffer_Send_Coord[iLocalVertexDonor][iDim] = donor_geometry->nodes->GetCoord(iPointDonor, iDim);
```

Comment:
```suggestion
        Buffer_Send_Coord(iLocalVertexDonor, iDim) = donor_geometry->nodes->GetCoord(iPointDonor, iDim);
```
The preferred syntax for su2matrix is (i, j) because this will check the range of each index in a debug build.
Operator [] returns a pointer, because we had to keep things compatible with numerics and so on.

Reply:
ok, changed in 7cc51eb